ball.js

```diff
@@ -0,0 +1,50 @@
+class Ball {
+  constructor(ctx, x, y, radius, color, speed) {
+    this.ctx = ctx;
+    this.x = x;
+    this.y = y;
+    this.radius = radius;
+    this.color = color;
+    this.speed = speed;
+    this.xDirection = 0;
```

Comment:
initial x direction is 0? shouldn't it be 1

Reply:
1 because game starts with ball dropping from top to bottom, x is constant until it hits paddle

Reply:
see comment below